golang/vaas/pkg/vaas/vaas_test.go

```diff
@@ -0,0 +1,387 @@
+package vaas
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+
+	"vaas/pkg/authenticator"
+	credentials "vaas/pkg/credentials"
+	"vaas/pkg/messages"
+	"vaas/pkg/options"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+)
+
+func setUp() Vaas {
+	godotenv.Load();
```

Comment:
We no not use `;` in Go :D

You could run `go fmt ./...` to format your whole project